client/cypress/support/commands.js

```diff
@@ -57,7 +57,9 @@ Cypress.Commands.add('logout', email => {
   cy.intercept('/auth/logout').as('logout')
   cy.intercept('/api/me').as('me')
   if (email) {
-    cy.findByText(email).click()
+    cy.findAllByText(email)
+      .eq(0)
```

Comment:
Had to add this because there can now be two elements on the page with the user email, the account button and the "Last edited by..." copy.

I tried `cy.findByRole('button', { name: email })`, but for whatever reason, that didn't work